docs/uml-class-diagrams/display02/production/DisplayComm2WayInterfaceImplementation/DisplayComm2WayInterfaceImplementation.puml

```diff
@@ -4,10 +4,11 @@ package edu.oakland.production.display02 {
     class DisplayComm2WayInterfaceImplementation {
         __
         .. Use Case 1 ..
+        {abstract} + LocationDataPoint passRfidRequest(int rfidNum)
```

Comment:
Can you please move this from the class `DisplayComm2WayInterfaceImplementation` to the interface  `DisplayComm2WayInterface`? Thank you!

Reply:
moved to DisplayComm2WayInterface. Thanks!